### PR TITLE
fix vuepress-plugin-mathjax can not support '\begin{aligned}' problem

### PR DESCRIPTION
**Pull request by liuxieric123**
vuepress-plugin-mathjax with dependency 'mathjax3' can not support just like '\begin{aligned}' expression. I have tried to change the dependency into 'mathjax-full', and it worked fine.
The chtml configure options need to add 'adaptive: false', to include the whole css file in mathjax.

### CODE REVIEW

**Comment by shigma** [comment]
@liuxieric123 很抱歉这么迟才回复你。

非常感谢你的 PR，但是我刚刚用 vuepress.github.io 试了一下好像渲染不出字的样子。你能再确认一下它可以用吗？

image

**Comment by Inlinebool** [comment]
你好，我在我的本地测试这个patch是能用的。不仅/begin{align}可以正确渲染，而且\tag和\label也能用了（patch之前\tag\label\eqref不能正常使用）。

**Comment by feibyte** [comment]
@Shigma I tested it locally and it worked well. What I did: I reran tests and updated snapshots, then changed file to HTML, it worked as expected.
Not quite sure what's the problem you had come across.
